<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test3</class>
 <widget class="QWidget" name="Test3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="line_name"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="com_sheets">
     <property name="toolTip">
      <string>所有工作表</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="but_addSheet">
     <property name="text">
      <string>创建工作表</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="com_state">
     <property name="toolTip">
      <string>工作表状态</string>
     </property>
     <item>
      <property name="text">
       <string>显示（SS_Visible）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>隐藏（SS_Hidden）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>绝对隐藏（SS_VeryHidden）</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>223</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
